source/js/modules/Modal.js

```diff
@@ -17,8 +17,8 @@ class ModalContainer {
 
     this.HtmlText = this._getHtmlText();
     this.rawText = this._getRawText();
-
-    if (!this.HtmlText || !this.rawText) throw new Error('[ModalContainer] >>> ModalContainer.constructor() >>> Object reference resulted in a nullptr');
+    
+    if (!this.HtmlText && !this.rawText) throw new Error('[ModalContainer] >>> ModalContainer.constructor() >>> Object reference resulted in a nullptr');
```

Comment:
if there was a mistake here, then maybe worth giving a comment explaining the intuition behind what is happening here

Reply:
It says clearly `Object reference resulted in a nullptr` aka `Object reference not set to an instance of an object.`

Reply:
Sorry, I wasn't clear enough. Why is "!this.HtmlText && !this.rawText" the condition? what are we intuitively checking, why do both of these need to be present? (answer in code please)

Reply:
No no, sorry it'a misleading but actually I'm checking whether both of them are empty... because we need at least one of them